<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KisWdgOptionsKranimseq</class>
 <widget class="QWidget" name="KisWdgOptionsKranimseq">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PNG Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QGridLayout" name="mainLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="startFrameLabel">
       <property name="text">
        <string>Start at frame:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="startFrameInput">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="keyFrameOnly">
       <property name="text">
        <string>Export only keyframes</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="stopFrameLabel">
       <property name="text">
        <string>Stop at frame</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="stopFrameInput">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
